config/initializers/color_themes.rb

```diff
@@ -0,0 +1,18 @@
+require 'pathname'
+
+# Generate the path to the .yml file with the available color themes.
+color_themes_file = Pathname.new(__FILE__).dirname.join('..').expand_path.join("color_themes.yml")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.